scripts/push-schema-changes.js

```diff
@@ -18,16 +26,25 @@ async function updateSchemaFile(repo) {
     targetBranch: "master",
     commitMessage: "Update metaphysics schema",
     body:
-      "Greetings human :robot: this PR was automatically created as part of metaphysics's deploy process",
+      "Greetings human :robot: this PR was automatically created as part of metaphysics' deploy process.",
     assignees: ["artsyit"],
     labels: ["Merge On Green"],
     update: (repoDir) => {
-      execSync(
-        `cp _schemaV2.graphql '${path.join(repoDir, "data/schema.graphql")}'`
-      )
+      const repoDest = path.join(repoDir, dest)
       execSync("yarn install --ignore-engines", { cwd: repoDir })
-      execSync("./node_modules/.bin/relay-compiler", { cwd: repoDir })
-      execSync("./node_modules/.bin/prettier --write data/schema.graphql", {
+      if (relay) {
```

Comment:
nonblocking: I'd probably have skipped the `relay` flag and done this by checking the file extension.

Reply:
I was wondering about that and didn't know whether we could assume the `.graphql`/`.json` association or not. Thanks!